types/ember__utils/index.d.ts

```diff
@@ -49,10 +49,12 @@ export function isPresent(obj?: any): boolean;
 export function tryInvoke<FNAME extends keyof T, T extends object>(
     obj: T,
     methodName: FNAME,
-    args: FunctionArgs<T[FNAME]>): T[FNAME] extends ((...args: any[]) => any)
-        ? ReturnType<T[FNAME]>
-        : undefined;
-export function tryInvoke<FNAME extends keyof T, T extends object>(obj: T, methodName: FNAME): T[FNAME] extends (() => any) ? ReturnType<T[FNAME]> : undefined;
+    args: FunctionArgs<T[FNAME]>,
+): T[FNAME] extends ((...args: any[]) => any) ? ReturnType<T[FNAME]> : undefined;
+export function tryInvoke<FNAME extends keyof T, T extends object>(
+    obj: T,
+    methodName: FNAME,
+): T[FNAME] extends (() => any) ? ReturnType<T[FNAME]> : undefined;
```

Comment:
Note for myself on Monday: revert this! it wasn't intended to make it in; auto-formatting got me.